src/components/About/Clients.module.css

```diff
@@ -0,0 +1,135 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+.container {
+  background: var(--dark-theme-background);
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 632px;
+  max-width: 100%;
+}
+.title {
+  color: #fff;
+  margin-bottom: 64px;
+}
+.containerImage {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100px;
+  transform: translate(0, -45%);
+}
+
+.news {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 3.5rem;
+}
+.logoVerge {
+  height: 1.5rem;
+  width: 8rem;
+}
+.logoJakarta {
+  height: 1.5rem;
+  width: 10rem;
+}
+.logoGuardian {
+  height: 1.5rem;
+  width: 10rem;
+}
+
+.logoTechRadar {
+  height: 2rem;
+  width: 10rem;
+}
+.logoGadgets {
+  height: 3rem;
+  width: 8rem;
+}
+
+@media (min-width: 768px) {
+  .container {
+    padding-inline: 40px;
+    height: 308px;
+    max-width: 100%;
+    overflow-x: hidden;
+  }
+  .title {
+    margin-bottom: 48px;
+  }
+  .news {
+    flex-direction: row;
+    gap: 3rem;
+    justify-content: space-between;
+  }
+
+  .logoVerge {
+    height: 1rem;
+    width: 6rem;
+  }
+  .logoJakarta {
+    height: 1rem;
+    width: 8rem;
+  }
+  .logoGuardian {
+    height: 1rem;
+    width: 8rem;
+  }
+
+  .logoTechRadar {
+    height: 1.5rem;
+    width: 8rem;
+  }
+  .logoGadgets {
+    height: 1.5rem;
+    width: 6rem;
+  }
+  .containerImage {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 200px;
+    width: 200px;
+    transform: translate(0, -45%);
+  }
+}
+
+@media (min-width: 960px) {
+  .containerImage {
+    position: absolute;
+    height: 150px;
+    width: 150px;
+    top: 0;
+    left: 0;
+    transform: translate(0, 0%);
+  }
+  .news {
+    flex-direction: row;
+    justify-content: space-between;
+  }
```

Comment:
These properties are already inherited from the `@media (min-width: 768px)` rules for `.news` (lines 65-69). If you reduce the gap for tablet sizes (see my other comment), you should set `gap: 3rem;` here though.

Reply:
I have kept it 2.5rem now as with initial 4rem it was overflowing, and as you said with 3rem , the tablet view is not correct. so 2.5rem for both tablet and desktop now looks good